tests/common/watch_test.go

```diff
@@ -0,0 +1,132 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.etcd.io/etcd/tests/v3/framework/config"
+	"go.etcd.io/etcd/tests/v3/framework/testutils"
+)
+
+func TestWatch(t *testing.T) {
+	testRunner.BeforeTest(t)
+	tcs := []struct {
```

Comment:
Why use different list of cluster setups than default?

Reply:
sure, updated.